x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -377,9 +380,12 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 		}
 	}
 
-	// Recalculate topic initial regret
-	if len(workersRegrets) > 0 {
-		updatedTopicInitialRegret, err := CalcTopicInitialRegret(
+	// Check if we have enough workers with enough experience to calculate the topic initial regret
+	// If we don't have enough workers with enough experience, use the fallback regrets
+	var updatedTopicInitialRegret alloraMath.Dec
+	var err error
+	if len(workersRegrets) >= 10 {
+		updatedTopicInitialRegret, err = CalcTopicInitialRegret(
```

Comment:
I think you need to pass in the regret at the `InitialRegretQuantile` not `InitialRegretQuantile` itself.